chill-java/src/main/java/com/twitter/chill/java/RegexSerializer.java

```diff
@@ -35,11 +35,10 @@ static public IKryoRegistrar registrar() {
     @Override
     public void write(Kryo kryo, Output output, Pattern pattern) {
         output.writeString(pattern.pattern());
-        output.writeInt(pattern.flags(), true);
     }
 
     @Override
     public Pattern read(Kryo kryo, Input input, Class<Pattern> patternClass) {
-        return Pattern.compile(input.readString(), input.readInt(true));
+        return Pattern.compile(input.readString());
```

Comment:
This seems the revert in #331; should we rebase?

Reply:
#331 is actually a breaking change since it appends an additional int to the serialized bytes. This is to revert that change.

Reply:
ahh ok 😄, i thought that was introduced in #326 and #331 was supposed to reverted it but closed due to build failure. Anyway good that it's reverted! 👍